src/views/CreateProfile.vue

```diff
@@ -193,30 +178,28 @@
             <li>Click <span class="highlited">Names</span>.</li>
             <li>
               On the <span class="highlited">Add New</span>
-              screen, type in your preferred name.
-              Note that the suffix .chain is automatically appended to your name.
-              This is mandatory in the naming system.
+              screen, type in your preferred name. Note that the suffix .chain is automatically
+              appended to your name. This is mandatory in the naming system.
             </li>
             <li>Click the <span class="highlited">+</span> button to proceed.</li>
             <li>Click Confirm.</li>
           </ul>
           <p>
             Only numbers and letters are allowed in .chain names.<br>
-            Names longer than 12 characters can be claimed by
-            the first to initiate a claim transaction for that particular name.<br>
+            Names longer than 12 characters can be claimed by the first to initiate a claim
+            transaction for that particular name.<br>
             Names 12 characters or shorter are subject to auction.<br>
             Shorter names are more expensive. Less characters mean longer auction times.<br>
             For more information on the naming system, click
             <a
               href="https://aeternity.com/documentation-hub/protocol/AENS/"
               target="_blank"
             >
-              here
-            </a>.
+              here </a>.
           </p>
           <p>
-            You can claim as many names on Superhero as you want.
-            But as a rule, you must have a minimum of XX AE in your wallet.
+            You can claim as many names on Superhero as you want. But as a rule, you must have a
+            minimum of XX AE in your wallet.
```

Comment:
XX ? should be specified

Reply:
Good question. I guess we should not limit this to specific amount as it will eventually depend on the length of the name that the user want to claim.

Reply:
I though about that, currently from what I understood you need to have at least 12 chars to claim from within the wallet which requires 2-3AE for it (or something like that) I think that might be the minimum we can put here. You will require more for shorter but you functionally need to use the auctions, which is not covered here. May be we can just mention that, so ppl are not confused

Reply:
I thought of just slightly modifying the text to not point to an exact value.

"... you must have enough AE to cover the price for the name ... "